README.md

```diff
@@ -147,6 +147,20 @@ You can pass more information using the command line:
 
     sigh resign ./path/app.ipa -i "iPhone Distribution: Felix Krause" -p "my.mobileprovision"
 
+# Manage
+
+With `sigh manage` you can list all provisioning profiles installed locally.
+
+    sigh manage
+
+Delete all expired provisioning profiles
+
+    sigh manage -e
+
+Or delete all `iOS Team Provisioning Profile` by using a regular expression
+
+    sigh manage -p "iOS\ {0,1}Team Provisioning Profile:"
```

Comment:
We could just do `iOS\ ?Team`

Depending on when they were generated by Xcode they are either called:
`iOS Team Provisioning Profile`
or
`iOSTeam Provisioning Profile`.